bitnami/harbor/templates/clair/clair-dpl.yaml

```diff
@@ -0,0 +1,89 @@
+{{ if .Values.clair.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: "{{ template "harbor.clair" . }}"
+  labels:
+    {{ include "harbor.labels" . | nindent 4 }}
+    app.kubernetes.io/component: clair
+spec:
+  replicas: {{ .Values.clair.replicas }}
+  selector:
+    matchLabels:
+      {{ include "harbor.matchLabels" . | nindent 6 }}
+      app.kubernetes.io/component: clair
+  template:
+    metadata:
+      labels:
+        {{ include "harbor.labels" . | nindent 8 }}
+        app.kubernetes.io/component: clair
+      annotations:
+        checksum/configmap: {{ include (print $.Template.BasePath "/clair/clair-cm.yaml") . | sha256sum }}
+      {{- if .Values.clair.podAnnotations }}
+        {{ toYaml .Values.clair.podAnnotations | nindent 8 }}
+      {{- end }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      containers:
+      - name: clair
+        image:  "{{ template "harbor.clairImage" . }}"
+        imagePullPolicy: {{ .Values.clairImage.pullPolicy | quote }}
+        {{- if .Values.clair.livenessProbe.enabled }}
+        livenessProbe:
+          httpGet:
+            path: /health
+            port: 6061
+          initialDelaySeconds: 30
+          periodSeconds: 10
+        {{- end }}
+        {{- if .Values.clair.readinessProbe.enabled }}
+        readinessProbe:
+          httpGet:
+            path: /health
+            port: 6061
+          initialDelaySeconds: 30
+          periodSeconds: 10
```

Comment:
Why are the readiness/liveness probes hardcoded? I'd use sth like:
```yaml
livenessProbe: {{ toYaml .Values. clair. livenessProbe | nindent 10 }}
readinessProbe: {{ toYaml .Values. clair.readinessProbe | nindent 10 }}
```

Reply:
They were already exposed in the values. I just update the yaml to get the values from there.